blocks/job-list/job-list.js

```diff
@@ -55,5 +55,7 @@ export default function decorate(block) {
 
   jobList.append(jobListings);
   block.textContent = '';
-  block.replaceWith(jobList);
+
+  // Replace wrapper div parent with new block.
+  block?.parentElement?.classList?.length === 0 ? block.parentElement.replaceWith(jobList) : block.replaceWith(jobList);;
```

Comment:
Got rid of these unnecessary extra wrapper divs, so I can make use of `:last-child` for the spacing adjustments.

Reply:
I've updated the `&:not(:last-child)` to be `&:has(~ .job-list)`, so it won't be thrown off by any content added at the end of that section in the future.